docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,253 +1,396 @@
+operations like creating users or reviewing Access Requests.
+
+Roles control access to infrastructure resources using **label matching**. All
+infrastructure resources that are enrolled in your Teleport cluster have
+**labels**, which are key-value pairs such as `env:dev`. When a user attempts to
+connect to a resource, the [Teleport
+Agent](../../enroll-resources/agents/agents.mdx) that proxies the resource
+checks the user's Teleport roles to ensure that the user has the appropriate
+permissions and, if not, denies the connection.
 
 ## Prerequisites
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
 - (!docs/pages/includes/tctl.mdx!)
 
-(!docs/pages/includes/permission-warning.mdx!)
-
-## Step 1/3. Add local users with preset roles
-
-Teleport provides several preset roles:
-
-(!docs/pages/includes/preset-roles-table.mdx!)
-
-<Tabs>
-<TabItem scope={["oss"]} label="Teleport Community Edition">
-Invite the local user Alice as cluster `editor`:
-
-```code
-$ tctl users add alice --roles=editor
-```
-</TabItem>
-<TabItem scope={["enterprise", "cloud"]} label="Commercial">
-Invite the local user Alice as cluster `editor` and `reviewer`:
-
-```code
-$ tctl users add alice --roles=editor,reviewer
-```
-</TabItem>
-
-</Tabs>
+- A Teleport user with permissions to create join tokens and other roles. We
+  recommend using a demo cluster with the preset `editor` role. 
+
+- Docker installed on your workstation. 
+
+  This guide illustrates how to register a server with Teleport using a Docker
+  container and the Teleport SSH Service. Docker is only required for the local
+  demo environment used in this guide. You can find installation instructions
+  for Docker on [Docker's website](https://docs.docker.com/get-docker/). If you
+  want to register servers in Teleport without using Docker, see the getting
+  started guide for [server
+  access](../../enroll-resources/server-access/getting-started.mdx).
+
+## Step 1/2. Enroll resources with labels
+
+In this section, you will enroll two servers with your Teleport cluster. One
+server will have the `env:local-dev` label and the other will have the
+`env:local-prod`.  Later in this guide, you will create a role that will allow a
+user to access a server with the `env:local-dev` label and deny access to the
+`env:local-prod` label.
+
+### Enroll a server with the `env:local-dev` label
+
+1. Create a join token for the server to use to join the cluster:
+
+   ```code
+   $ tctl tokens add --type=node
+   ```
+
+1. Assign the token to <Var name="token" /> and the host and web port of your
+   Teleport Proxy Service to <Var name="example.teleport.sh:443" />.
+
+1. Copy the following Teleport configuration to a file called `local-dev.yaml`:
+
+   ```yaml
+   version: v3
+   teleport:
+     data_dir: /var/lib/teleport
+     join_params:
+       token_name: <Var name="token" />
+       method: token
+     proxy_server: <Var name="example.teleport.sh:443" />
+   auth_service:
+     enabled: "no"
```

Comment:
I think we just merged a PR to change these yes/no values to true/false. Let's not undo those changes by adding more of the old style.